src/pages/[componentAccountId]/widget/[componentName].tsx

```diff
@@ -27,6 +26,7 @@ const ViewComponentPage: NextPageWithLayout = () => {
   }, [router.query]);
 
   useEffect(() => {
+    console.log("checking for zendesk widget");
```

Comment:
I wonder why I can't see this log on local? @calebjacob I think it might me relative to this comment https://github.com/near/near-discovery/pull/239/files#r1223205744 (made by @charleslavon )

Reply:
Ok, I got the thing